lib/common.sh

```diff
@@ -24,17 +24,19 @@ export PROG
 
 set -o errtrace
 
-# OSX not supported.  Tell them right away
-if [[ $(uname) == "Darwin" ]]; then
-  echo "OSX is not a supported OS for running these tools. Exiting..."
-  exit 1
-fi
-
 ##############################################################################
 # COMMON CONSTANTS
 #
-TOOL_LIB_PATH=${TOOL_LIB_PATH:-$(dirname $(readlink -ne $BASH_SOURCE))}
-TOOL_ROOT=${TOOL_ROOT:-$(readlink -ne $TOOL_LIB_PATH/..)}
+
+if [[ $(uname) == "Darwin" ]]; then
+  # Support for OSX.
+  TOOL_LIB_PATH=${TOOL_LIB_PATH:-$(dirname $(greadlink -f $BASH_SOURCE))}
```

Comment:
is `greadlink` GNU `readlink`? if so, shouldn't the args be the same for both OSX and non-OSX?

Reply:
right. but why are we doing `greadlink -f` here but `readlink -ne` below?

Reply:
Oh, gotcha. Changed them to the same - PTAL.